src/main/java/frc/robot/commands/drive/SwerveDrive.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands.drive;
+
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.Constants.DriveConstants;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class SwerveDrive extends CommandBase{
```

Comment:
This command name could be easily confused with a subsystem. Maybe name this "SwerveDriveCommand" or something that clearly indicates that it is not the Swerve Drive itself.